src/main/scala/is/hail/expr/Relational.scala

```diff
@@ -141,7 +141,7 @@ case class MatrixValue(
       FileFormat.version.rep,
       hc.version,
       "../references",
-      typ.rowsTableType,
+      TableType(typ.entriesRVType, None, typ.globalType),
```

Comment:
was this a separate bug?

Reply:
Yes, although the entries table is never loaded directly and we don't use that type in MatrixRead.

Reply:
ahh okay